packages/trader/src/Modules/CFD/Containers/compare-accounts-content.tsx

```diff
@@ -0,0 +1,690 @@
+import React from 'react';
+import { Table, Text, ThemedScrollbars, Div100vhContainer } from '@deriv/components';
+import { localize, Localize } from '@deriv/translations';
+import { isDesktop, CFD_PLATFORMS, isLandingCompanyEnabled, PlatformContext } from '@deriv/shared';
+import { LandingCompany } from '@deriv/api-types';
+
+type TCFDAttributeDescriberProps = {
+    name: string;
+    counter: number | null;
```

Comment:
Type "number" is a part of "null" too, please add a correct type for this

Reply:
No changes needed as we need `null` type to differentiate the counter for `footnote`